src/main/java/io/xpring/xrpl/DefaultXpringClient.java

```diff
@@ -37,11 +33,9 @@
     /**
      * No-args Constructor.
      */
-    public DefaultXpringClient() {
+    public DefaultXpringClient(String grpcURL) {
         this(ManagedChannelBuilder
-                .forTarget(XPRING_GRPC_URL)
-                // Let's use plaintext communication because we don't have certs
-                // TODO: Use TLS!
+                .forTarget(grpcURL)
                 .usePlaintext()
```

Comment:
Does this TODO that you removed still apply? Or should we always use plaintext communication?

Reply:
We assume, at least until if/when rippled admin methods are implemented, that we're going to defer TLS. 

The reason is that these calls are doing read / writes to a public ledger and so we weren't overly concerned. There's a somewhat nebulous problem about how to get certs into a rippled node as well, which MAYBE websockets have handled, but we haven't dealt with. 

There's an [asana task](https://app.asana.com/0/1143722888069470/1145001263710841) tracking this and this TODO was written on day one when we were like 'sure we can probably write a java sdk in a few hours', so I had kinda assumed it was redundant. 

Happy to revert and take ownership if it bugs you.

Reply:
I think it's fine. I assume we sign transactions locally/offline?

Reply:
Yes, that is correct. The on-node signing API is deprecated and isn't particularly secure. 

We use ripple-keypairs to do this signing.